### DISCUSSION TITLE
Hide host console window

### DISCUSSION THREAD

**Discussion by BctfN0HUK7Yg** [emilk/egui#1003] [Q&A]
I use eframe template.
When I compile my app with "cargo build --release" I have 2 windows. One is my app, and another is "host console window". But when I compile without "--release". I have only my app window.

How to make release with only one window?

**Comment by TicClick** [comment]
use the [`windows_subsystem` top-level crate attribute](https://rust-lang.github.io/rfcs/1665-windows-subsystem.html#detailed-design) by adding this line to your `main.rs`:

```rust
#![windows_subsystem = "windows"]
```

**Reply by emilk** [reply]
Interesting! I guess I should add `#![windows_subsystem = "windows"]` to all egui/eframe examples (including `eframe_template`).

**Reply by TicClick** [reply]
probably, but it's worth noting this comes with losing access to the app's STDOUT — you won't see any `println!`s and probably `panic!`s as well
